src/components/DictionaryManageDialog.vue

```diff
@@ -0,0 +1,777 @@
+                active-class="active-word"
+              >
+                <q-item-section>
+                  <q-item-label class="text-display">{{
+                    value.surface
+                  }}</q-item-label>
+                  <q-item-label caption>{{ value.yomi }}</q-item-label>
+                </q-item-section>
+              </q-item>
+            </q-list>
+            <div class="q-ma-sm">
+              <q-btn
+                outline
+                text-color="display"
+                class="text-no-wrap text-bold new-word-button"
+                @click="discardOrNotDialog(newWord)"
+                :disable="uiLocked"
+                >新規登録</q-btn
+              >
+            </div>
+          </div>
+          <div class="col-8 no-wrap text-no-wrap">
+            <div class="row q-pl-md q-mt-md">
+              <div class="text-h6">単語</div>
+              <form @submit.stop="yomiInput.focus()">
+                <q-input
+                  ref="surfaceInput"
+                  class="word-input"
+                  v-model="surface"
+                  @blur="setSurface(surface)"
+                  dense
+                  :disable="uiLocked"
+                />
+              </form>
+            </div>
+            <div class="row q-pl-md q-pt-sm">
+              <div class="text-h6">読み</div>
+              <form @submit.stop="yomiInput.blur()">
+                <q-input
+                  ref="yomiInput"
+                  class="word-input"
+                  v-model="yomi"
+                  @blur="setYomi(yomi)"
+                  dense
+                  :error="!isOnlyHiraOrKana"
+                  :disable="uiLocked"
+                >
+                  <template v-slot:error>
+                    読みに使える文字はひらがなとカタカナのみです。
+                  </template>
+                </q-input>
+              </form>
+            </div>
+            <div class="row q-pl-md q-pt-sm text-h6">アクセント調整</div>
+            <div class="row q-pl-md accent-desc">
+              語尾のアクセントを考慮するため、「は(ワ)」が自動で挿入されます。
+            </div>
+            <div class="row q-px-md" style="height: 130px">
+              <div class="play-button">
+                <q-btn
+                  v-if="!nowPlaying && !nowGenerating"
+                  fab
+                  color="primary-light"
+                  text-color="display-dark"
+                  icon="play_arrow"
+                  @click="play"
+                />
+                <q-btn
+                  v-else
+                  fab
+                  color="primary-light"
+                  text-color="display-dark"
+                  icon="stop"
+                  @click="stop"
+                  :disable="nowGenerating"
+                />
+              </div>
+              <div
+                ref="accentPhraseTable"
+                class="accent-phrase-table overflow-hidden-y"
+                :style="[centeringAccentPhrase && 'justify-content: center']"
```

Comment:
文法的に正しくないらしいです（falseが入れられない）
```suggestion
                :style="{
                  justifyContent: centeringAccentPhrase ? 'center' : undefined,
                }"
```